### PR TITLE
No colorization for history and compare windows

### PR DESCRIPTION
**Pull request by dehilsterlexis**

### CODE REVIEW

**Comment by richardkchapman** [comment]
Needs rebasing

**Comment by dehilsterlexis** [comment]
@richardkchapman yes. I left it like this knowing I needed to rebase. Should it be on 6.4.2 or 6.4.4?

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

File: wlib/SourceView.cpp [hunk 1]
```diff
@@ -469,6 +458,29 @@ void CSourceCtrl::DoInit()
     SetFoldFlags(SC_FOLDFLAG_LINEAFTER_CONTRACTED);
 }
 
+void CSourceCtrl::InitLanguage(IAttributeType *attrType, BOOL clearSourceType)
```
Comment (GordonSmith) [inline wlib/SourceView.cpp]: Do you ever set attrType and have clearSourceType=false?

Comment (dehilsterlexis) [reply wlib/SourceView.cpp]: Yes. It is false as a default in the call in SourceView.cpp on line 392.

Comment (dehilsterlexis) [reply wlib/SourceView.cpp]: No. I thought of the same thing. But I usually leave the option open. Could go either way.

**Comment by GordonSmith** [comment]
Lets remove the clearSourceType and always clear it when attrType != NULL?

**Comment by dehilsterlexis** [comment]
Agreed

**Comment by dehilsterlexis** [comment]
@GordonSmith changed and waiting review.

**Comment by richardkchapman** [comment]
This should have referenced a Jira...

**Comment by dehilsterlexis** [comment]
@richardkchapman sorry about that! Put it in the title.

**Pull request merged**